backend/src/main/resources/application-production.properties

```diff
@@ -1,17 +1,7 @@
-#datasource configurations
-spring.datasource.url=jdbc:mysql://mysql.agh.edu.pl:3306/pmakarew?useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC 
-spring.datasource.username=pmakarew
-spring.datasource.password=TyRDnb0C58H7S4E4
+spring.datasource.url=jdbc:mysql://server531716.nazwa.pl/io_nk_production?useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC 
+spring.datasource.username=io-nk
+spring.datasource.password=.aa<~T6Vm5W~",a`
```

Comment:
Możesz chcieć zrobić github secrets, bo teraz każdy może sobie wejść na tego githuba, zobaczyć to hasło z kodu i porwać tą bazę danych

Oczywiście jeżeli ci się chce, mi się nie chciało dla starej bazy